<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>718</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>236</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>196</red>
        <green>196</green>
        <blue>196</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>78</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>105</green>
        <blue>105</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>206</green>
        <blue>206</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>236</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>196</red>
        <green>196</green>
        <blue>196</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>78</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>105</green>
        <blue>105</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>206</green>
        <blue>206</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>78</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>236</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>196</red>
        <green>196</green>
        <blue>196</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>78</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>105</green>
        <blue>105</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>78</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>78</red>
        <green>78</green>
        <blue>78</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>157</red>
        <green>157</green>
        <blue>157</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titre">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Test unitaire de la classe ModBus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connexion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ventilateurs_on">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>240</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Température</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deconnexion">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;
color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Niveau</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>370</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:24px;
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>Ventilateurs :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ventilateurs_off">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extracteurs_off">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extracteurs_on">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>Extracteurs :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pluie_off">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pluie_on">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>Pluie</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>Natte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="natte_on">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="natte_off">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>Lampe IR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ir_on">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ir_off">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>Lampe UV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uv_on">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uv_off">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depart">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Départ Cycle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arret">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Arret Cycle</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>271</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16px;&quot;&gt;172.16.150.3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>480</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="port_button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>540</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camera_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Caméra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="balance_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="port_button_close">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>610</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fermer port</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_cam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px;
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_balance">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px;
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_etat_timer">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 12px;</string>
    </property>
    <property name="text">
     <string>Etat Timer :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bdd_archivage">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>370</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>BDD MySQL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
